robot/impl/local_robot.go

```diff
@@ -1435,3 +1444,44 @@ func (r *localRobot) MachineStatus(ctx context.Context) (robot.MachineStatus, er
 func (r *localRobot) Version(ctx context.Context) (robot.VersionResponse, error) {
 	return robot.Version()
 }
+
+func (r *localRobot) checkMaintenanceSensor(newConfig *config.Config) (bool, error) {
+	if newConfig.MaintenanceConfig == nil {
+		return true, errors.New("maintenanceConfig undefined. Using default reconfigure")
+	}
```

Comment:
move this up to where this is called and don't alert users if this does not exist